math/int.go

```diff
@@ -442,6 +442,10 @@ var (
 )
 
 func (i *Int) Size() (size int) {
+	if i == nil || i.i == nil {
```

Comment:
@julienrbrt wondering where the "return 1" comes from?

Reply:
I checked the result of `(&math.Int{}).Size()` on math/v1.0.1